dotnet/samples/KernelSyntaxExamples/Getting_Started/Step9_Safe_Chat_Prompts.cs

```diff
@@ -0,0 +1,245 @@
+using System.Threading.Tasks;
+using Examples;
+using Microsoft.SemanticKernel;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GettingStarted;
+
+public sealed class Step9_Safe_Chat_Prompts(ITestOutputHelper output) : BaseTest(output)
+{
+    /// <summary>
+    /// Show how to construct a chat prompt safely and invoke it.
+    /// </summary>
+    [Fact]
+    public async Task RunAsync()
+    {
+        // Create a logging handler to output HTTP requests and responses
+        var handler = new LoggingHandler(new HttpClientHandler(), this.Output);
+        var client = new HttpClient(handler);
+
+        // Create a kernel with OpenAI chat completion
+        Kernel kernel = Kernel.CreateBuilder()
+            .AddOpenAIChatCompletion(
+                modelId: TestConfiguration.OpenAI.ChatModelId,
+                apiKey: TestConfiguration.OpenAI.ApiKey,
+                httpClient: client)
+            .Build();
+
+        // Each example demonstrates a different way to construct a chat prompt
+        await ExamplePlainTextAsync(kernel);
+        await ExampleTextContentAsync(kernel);
+        await ExampleHtmlEncodedTextAsync(kernel);
+        await ExampleCDataSectionAsync(kernel);
+        await ExampleEmptyInputVariableAsync(kernel);
+        await ExampleSafeInputVariableAsync(kernel);
+        await ExampleUnsafeInputVariableAsync(kernel);
+        await ExampleSafeFunctionAsync(kernel);
+        await ExampleUnsafeFunctionAsync(kernel);
+        await ExampleTrustedVariablesAsync(kernel);
+        await ExampleTrustedFunctionAsync(kernel);
+        await ExampleTrustedTemplateAsync(kernel);
+    }
+
+    private async Task ExampleTrustedTemplateAsync(Kernel kernel)
+    {
+        KernelFunction trustedMessageFunction = KernelFunctionFactory.CreateFromMethod(() => "<message role=\"system\">You are a helpful assistant who knows all about cities in the USA</message>", "TrustedMessageFunction");
+        KernelFunction trustedContentFunction = KernelFunctionFactory.CreateFromMethod(() => "<text>What is Seattle?</text>", "TrustedContentFunction");
+        kernel.ImportPluginFromFunctions("TrustedPlugin", [trustedMessageFunction, trustedContentFunction]);
+
+        var chatPrompt = @"
+            {{TrustedPlugin.TrustedMessageFunction}}
+            <message role=""user"">{{$input}}</message>
+            <message role=""user"">{{TrustedPlugin.TrustedContentFunction}}</message>
+        ";
+        var promptConfig = new PromptTemplateConfig(chatPrompt);
+        var kernelArguments = new KernelArguments()
+        {
+            ["input"] = "<text>What is Washington?</text>",
+        };
+        var factory = new KernelPromptTemplateFactory() { AllowUnsafeContent = true };
+        var function = KernelFunctionFactory.CreateFromPrompt(promptConfig, factory);
+        WriteLine(await RenderPromptAsync(promptConfig, kernel, kernelArguments, factory));
+        WriteLine(await kernel.InvokeAsync(function, kernelArguments));
```

Comment:
@markwallace-microsoft I would probably add following things to PR description:
- This example, that shows how to easily return to previous behaviour. 
- Highlighted message that with encoding by default, the token usage will increase, since encoded prompt takes more symbols.